code/datums/emergency_calls/emergency_call.dm

```diff
@@ -0,0 +1,217 @@
+//This file deals with distress beacons. It randomizes between a number of different types when activated.
+//There's also an admin commmand which lets you set one to your liking.
+
+
+//The distress call parent.
+/datum/emergency_call
+	var/name = ""
+	var/mob_max = 10
+	var/mob_min = 1
+	var/landmark_tag
+	var/dispatch_message = "An encrypted signal has been received from a nearby vessel. Stand by." //Message displayed to marines once the signal is finalized.
```

Comment:
A specific requirement here:

We need any messages about requesting/dispatching emergency teams, to be shown to all crewmembers, but NOT to be shown to necromorphs and signals